products/firefox_iOS/adding_new_iOSlocale.md

```diff
@@ -0,0 +1,43 @@
+# Adding a new locale to Firefox for iOS
+
+## Prerequisites
+
+### iOS language support
+
+iOS currently supports [two tiers of locales](https://people-mozilla.org/~sarentz/ios-locales.html), and a locale must be part of that list in order to ship on Firefox for iOS. In fact, there is no locale switcher on Firefox for iOS (vs Firefox on Android, which does have one) - so if a locale is supported totally depends on what languages the system supports itself.
```

Comment:
I think the second sentence repeats what is already in the first part of the paragraph. I would rephrase the second part as : "Unlike Firefox for Android, there’s no locale switcher included in the app, so it’s possible to ship only locales that are supported by iOS.".